packages/flutter/test/rendering/debug_test.dart

```diff
@@ -15,10 +15,10 @@ void main() {
     final Matrix4 identity = Matrix4.identity();
     final List<String> description = debugDescribeTransform(identity);
     expect(description, equals(<String>[
-      '[0] 1.0,0.0,0.0,0.0',
-      '[1] 0.0,1.0,0.0,0.0',
-      '[2] 0.0,0.0,1.0,0.0',
-      '[3] 0.0,0.0,0.0,1.0',
+      ignoreWebNumericQuirks('[0] 1.0,0.0,0.0,0.0'),
```

Comment:
What is ignoreWebNumericQuirks?

Reply:
Added the implementation here: https://github.com/flutter/flutter/pull/33408

Reply:
Alternatively, we could implement `debugDescribeTransform` such that it produces identical formatting. This is debug code anyway, so it won't affect performance.

Reply:
Seems like we could solve this by finally fixing https://github.com/flutter/flutter/issues/1221

Reply:
I spoke with @ferhatb today and they already have code that needs to be upstreamed to fix this

Reply:
Err, this particular case and not the more general isssue :)